package.json

```diff
@@ -2,8 +2,9 @@
     "name": "tabsoutliner", 
     "license": "MPL 2.0", 
     "author": "Bernard `Guyzmo` Pratz", 
-    "version": "0.4", 
+    "version": "0.0.4", 
```

Comment:
urgh, ok, then make it `0.5.0`, as it's still a change from the former
jetpack dev scheme.